styles/_settings.scss

```diff
@@ -27,9 +27,9 @@ $cloudy-blue: #aadad1;
 $light-grey-blue: #8db6ca;
 $warm-pink: #f65e6e;
 
-$link-green: #bcda59;
-$link-blue: #2f7fab;
-$link-pink: $warm-pink;
+$qual-soybeans: #bcda59;
+$qual-soybeancake: #2f7fab;
+$qual-soybeanoil: $warm-pink;
```

Comment:
Why do you create another colour variable when you already have the `$warm-pink` one?